src/include/function/map/operations/map_operation.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <cassert>
+#include <cstring>
+
+#include "common/vector/value_vector.h"
+#include "common/vector/value_vector_utils.h"
+
+namespace kuzu {
+namespace function {
+namespace operation {
+
+struct Map {
+    static void operation(common::list_entry_t& keyEntry, common::list_entry_t& valueEntry,
```

Comment:
I wonder if we should move implementation to cpp files

Reply:
I think we write all operation definitions inside .h file instead of .cpp file